### PR TITLE
 Fixing broken xrefs to anchors and many other improvements

### PR DESCRIPTION
**Pull request by mmattel**
- I found a new and really useful broken linkchecker (will write a own PR to add it to blc-docs).
See: https://github.com/filiph/linkcheck
Using that one, I identified a huge amount of broken references to anchors which are now fixed :smile:
The process was: identification, finding the reference, fixing the reference, testing with yarn antora in cycles
- When there was an example reference like http://example..., this caused a lot of blc issues.
I added a backslash where suitable in front of that link which lets antora render this as text and not as link.
- Fixed some external broken links
- Changed many keyboard, button and menu entries to their proper attributes
- The configuration/encryption files have not been added to the navigation
These are now added in a new section in configuration
- In lists, added a paragraph continuation attribute so that pictures or notes ect belonging to that list item are rendered in correct alignment
- Some text improvements

### CODE REVIEW

File: modules/admin_manual/nav.adoc [hunk 1]
```diff
@@ -155,6 +173,9 @@
 *** User Management
 **** xref:enterprise/user_management/user_auth_shibboleth.adoc[Shibboleth Integration]
 **** xref:enterprise/user_management/saml_2.0_sso.adoc[SAML 2.0 Based SSO]
-** xref:document_classification/index.adoc[Document Classification]
```
Comment (settermjd) [inline modules/admin_manual/nav.adoc]: Why this change?

File: modules/admin_manual/pages/configuration/files/encryption/enabling-user-key-encryption.adoc [hunk 2]
```diff
@@ -57,7 +58,7 @@ sudo -u www-data php occ maintenance:singleuser --off
 ....
 
 [[how-to-enable-users-file-recovery-keys]]
-== How To Enable User File Recovery Keys
+== How To Enable Users File Recovery Keys
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/files/encryption/enabling-user-key-encryption.adoc]: This change doesn't read correctly to me.

File: modules/admin_manual/pages/configuration/files/encryption/master-key-encryption.adoc [hunk 3]
```diff
@@ -2,21 +2,20 @@
...
+:toclevels: 1
 :experimental:
 :keywords: master key based encryption, encryption
 :description: This guide will show you how to work with Master Key encryption in ownCloud. 
 
 == Introduction
 
-With Master Key based encryption, there is only one key (or key pair) and all files are encrypted using that key pair. 
-This is highly recommended for new instances to avoid restrictions in functionality of user key encryption.
+With Master Key based encryption, there is only one key (or key pair) and all files are encrypted
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/files/encryption/master-key-encryption.adoc]: Is there a specific need to split sentences over multiple lines?

File: modules/admin_manual/pages/configuration/files/external_storage/dropbox.adoc [hunk 4]
```diff
@@ -49,9 +51,11 @@ http(s)://<<Server_Address>>/index.php/settings/personal?sectionid=storage
 ....
 
 [[dropbox-install-step-three]]
-== Step Three - Create a Dropbox Share
-To create a Dropbox share, under `admin -> Settings -> Admin -> Storage`, check the "Enable external storage" checkbox, if it’s not already checked.
-Then, in the drop-down list under "External storage", click the first "Dropbox" option.
+== Create a Dropbox Share
+
+To create a Dropbox share, under menu:Admin[Settings > Storage],
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/files/external_storage/dropbox.adoc]: Is there a specific need to split sentences over multiple lines?

File: modules/admin_manual/pages/configuration/files/external_storage/ftp.adoc [hunk 5]
```diff
@@ -18,14 +18,16 @@ To connect to an FTP server, you will need:
 Your new mountpoint is available to all users by default, and you may restrict access by entering specific users or groups in the *Available for* field.
 
 Optionally, ownCloud can use FTPS (FTP over SSL) by checking `Secure ftps://`. 
-This requires additional configuration with your root certificate, if the FTP server uses https://en.wikipedia.org/wiki/Self-signed_certificate[a self-signed certificate].
+This requires additional configuration with your root certificate, if the FTP server uses
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/files/external_storage/ftp.adoc]: Is there a specific need to split sentences over multiple lines?

File: modules/admin_manual/pages/configuration/files/external_storage/ftp.adoc [hunk 6]
```diff
@@ -18,14 +18,16 @@ To connect to an FTP server, you will need:
...
+https://en.wikipedia.org/wiki/Self-signed_certificate[a self-signed certificate].
 See xref:configuration/server/import_ssl_cert.adoc[Importing System-wide and Personal SSL Certificates] for more information.
 
 image:configuration/files/external_storage/ftp.png[ownCloud GUI FTP configuration.]
 
 The external storage `FTP/FTPS` needs the `allow_url_fopen` PHP setting to be set to `1`.
 When having connection problems make sure that it is not set to `0` in your `php.ini`.
-See https://doc.owncloud.com/server/latest/admin_manual/issues/general_troubleshooting.html#label-phpinfo[PHP Version and Information] to learn how to find the right `php.ini` file to edit.
+See xref:configuration/general_topics/general_troubleshooting.adoc#php-version-and-information[PHP Version and Information]
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/files/external_storage/ftp.adoc]: Is there a specific need to split sentences over multiple lines?

File: modules/admin_manual/pages/document_classification/index.adoc [hunk 7]
```diff
@@ -232,15 +224,15 @@ Assume you want to automatically classify all PDF documents uploaded by users th
...
 * Within user management create the group "_Management_" and add some users.
-* Navigate to the "_Settings_" section "_Workflows & Tags_".
+* Navigate to menu:Settings[Workflows & Tags].
 * In the xref:enterprise/file_management/files_tagging.adoc#tag-manager[Collaborative Tags Management] panel, create a tag of type "_Static_" and call it `Class: Confidential`.
   Adding a group with special privileges for the tag is optional.
-* In the panel "_Workflow_" you can now set up the classification rules. Hit "_Add new workflow_" and specify a useful name.
+* In the panel "_Workflow_" you can now set up the classification rules. Hit btn:[Add new workflow] and specify a useful name.
   Now configure the conditions that trigger the classification once they are met.
-  For that choose "_User group_" from the drop-down menu, hit **+**, then choose "_File mimetype_" and hit **+** again.
+  For that choose "_User group_" from the drop-down menu, click `+`, then choose "_File mimetype_" and click `+` again.
```
Comment (settermjd) [inline modules/admin_manual/pages/document_classification/index.adoc]: How about the `btn:[]` markup?

File: modules/admin_manual/pages/installation/manual_installation.adoc [hunk 8]
```diff
@@ -379,7 +370,10 @@ long as it sends the correct `X-Forwarded-Host` header.
 
 NOTE: For further information on improving the quality of your ownCloud installation, please see xref:installation/configuration_notes_and_tips.adoc[the configuration notes and tips guide].
 
-NOTE: Admins of SELinux-enabled distributions such as _CentOS_, _Fedora_, and _Red Hat Enterprise Linux_ may need to set new rules to enable installing ownCloud. See xref:installation/configuration_notes_and_tips.adoc#selinux[SELinux] for a suggested configuration.
+NOTE: Admins of SELinux-enabled distributions such as _CentOS_, _Fedora_, and
```
Comment (settermjd) [inline modules/admin_manual/pages/installation/manual_installation.adoc]: Is there a specific need to split sentences over multiple lines?

File: modules/admin_manual/nav.adoc (hunk 1 above)
Comment (mmattel) [reply modules/admin_manual/nav.adoc]: very easy,
beause this was the only item not having a drop down
also trouble shooting with a single entry has a drop down
harmonizing layout

File: modules/admin_manual/pages/configuration/files/encryption/master-key-encryption.adoc (hunk 3 above)
Comment (mmattel) [reply modules/admin_manual/pages/configuration/files/encryption/master-key-encryption.adoc]: yes, unreadbility in editor,
you have to move editor slider to read the whole text
awful

File: modules/admin_manual/pages/configuration/files/external_storage/dropbox.adoc (hunk 4 above)
Comment (mmattel) [reply modules/admin_manual/pages/configuration/files/external_storage/dropbox.adoc]: see above

File: modules/admin_manual/pages/configuration/files/external_storage/ftp.adoc (hunk 5 above)
Comment (mmattel) [reply modules/admin_manual/pages/configuration/files/external_storage/ftp.adoc]: see above

File: modules/admin_manual/pages/configuration/files/external_storage/ftp.adoc (hunk 6 above)
Comment (mmattel) [reply modules/admin_manual/pages/configuration/files/external_storage/ftp.adoc]: see above

File: modules/admin_manual/pages/document_classification/index.adoc (hunk 7 above)
Comment (mmattel) [reply modules/admin_manual/pages/document_classification/index.adoc]: good catch, will do

File: modules/admin_manual/pages/installation/manual_installation.adoc (hunk 8 above)
Comment (mmattel) [reply modules/admin_manual/pages/installation/manual_installation.adoc]: see above

File: modules/admin_manual/pages/configuration/files/encryption/master-key-encryption.adoc (hunk 3 above)
Comment (settermjd) [reply modules/admin_manual/pages/configuration/files/encryption/master-key-encryption.adoc]: Which editor are you using?

Comment (settermjd) [reply modules/admin_manual/pages/configuration/files/encryption/master-key-encryption.adoc]: Can you post a screenshot?

File: modules/admin_manual/pages/configuration/files/encryption/enabling-user-key-encryption.adoc (hunk 2 above)
Comment (mmattel) [reply modules/admin_manual/pages/configuration/files/encryption/enabling-user-key-encryption.adoc]: Will check if the anchor is used somewhere and change it accordingly

File: modules/admin_manual/nav.adoc (hunk 1 above)
Comment (settermjd) [reply modules/admin_manual/nav.adoc]: I don't quite understand the second point. Can you further clarify?

File: modules/admin_manual/pages/configuration/files/encryption/enabling-user-key-encryption.adoc (hunk 2 above)
Comment (settermjd) [reply modules/admin_manual/pages/configuration/files/encryption/enabling-user-key-encryption.adoc]: The anchor's not the issue, it's the change of "User" to "Users". I don't believe that this is grammatically correct in this context (but will double-check).

Comment (settermjd) [reply modules/admin_manual/pages/configuration/files/encryption/enabling-user-key-encryption.adoc]: After checking it, it's grammatically correct, however I don't see the need for the change.

File: modules/admin_manual/nav.adoc (hunk 1 above)
Comment (mmattel) [reply modules/admin_manual/nav.adoc]:
```
** Document Classification
*** xref:document_classification/index.adoc[Document Classification and Policy Enforcement]

** Troubleshooting
*** xref:troubleshooting/providing_logs_and_config_files.adoc[Retrieve Log Files and Configuration Settings]
```

File: modules/admin_manual/pages/configuration/files/encryption/master-key-encryption.adoc (hunk 3 above)
Comment (mmattel) [reply modules/admin_manual/pages/configuration/files/encryption/master-key-encryption.adoc]: image

File: modules/admin_manual/pages/document_classification/index.adoc (hunk 7 above)
Comment (mmattel) [reply modules/admin_manual/pages/document_classification/index.adoc]: fixed

File: modules/admin_manual/nav.adoc (hunk 1 above)
Comment (mmattel) [reply modules/admin_manual/nav.adoc]: image

File: modules/admin_manual/pages/configuration/files/encryption/master-key-encryption.adoc (hunk 3 above)
Comment (settermjd) [reply modules/admin_manual/pages/configuration/files/encryption/master-key-encryption.adoc]: screenshot
